OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -145,21 +145,26 @@ const std::string LiveFunctionsDataView::MENU_ACTION_JUMP_TO_MIN =
     "Jump to min";
 const std::string LiveFunctionsDataView::MENU_ACTION_JUMP_TO_MAX =
     "Jump to max";
+const std::string LiveFunctionsDataView::MENU_ACTION_DISASSEMBLY =
+    "Go to Disassembly";
 
 //-----------------------------------------------------------------------------
 std::vector<std::string> LiveFunctionsDataView::GetContextMenu(
     int a_ClickedIndex, const std::vector<int>& a_SelectedIndices) {
   bool enable_select = false;
   bool enable_unselect = false;
+  bool enable_disassembly = false;
```

Comment:
Maybe set enable_disassembly once? enable_disassembly = !a_SelectedIndices.empty();

Reply:
Thanks. I adjusted the code accordingly.